phpstan.dist.neon

```diff
@@ -0,0 +1,16 @@
+parameters:
+    level: 1
+    paths:
+        - src
+    excludePaths:
+        - src/Behat/Testwork/EventDispatcher/TestworkEventDispatcherSymfonyLegacy.php
```

Comment:
We exclude these two files as they generate a lot of false positives depending on the Symfony version